packages/ffe-accordion-react/src/AccordionItem.js

```diff
@@ -49,10 +49,7 @@ const AccordionItem = ({
             })}
             {...rest}
         >
-            <Heading6
-                className="ffe-accordion-item__heading"
-                aria-level={headingLevel}
-            >
+            <AccordionItemHeader headingLevel={headingLevel}>
```

Comment:
Kanskje du kan se på [Heading i ffe-core-react](https://github.com/SpareBank1/designsystem/blob/develop/packages/ffe-core-react/src/typography/Heading.js#L20-L38) for inspirasjon hvordan dette kan løses uten en lang switch-case som i AccordionItemHeader.